run/doc.go

```diff
@@ -0,0 +1,6 @@
+/*
+Package run provides tools for running solvers. When `run.Run` is called by your
+application, the runner configured by the environment variable `NEXTMV_RUNNER`
+will be used. Valid values are `cli` (default) and `http`.
+*/
+package run
```

Comment:
There is already a description in `run.go`:

```go
// Package run provides tools for running solvers.
package run
```

If we change the README to point to the pkg docs, we can delete this file. If you want to keep it, then I would delete the package comment from the `run.go` file and just keep this one.